pkg/policy/mapstate.go

```diff
@@ -437,23 +547,23 @@ func (keys MapState) insertIfNotExists(key Key, entry MapStateEntry) {
 //
 // The above can be accomplished by:
 //
-// 1. Change existing L4-only ALLOW key on matching port that does not already
-//    redirect to redirect.
-//    - e.g., 0:80=allow,0 -> 0:80=allow,<proxyport>
-// 2. If allow-all policy exists, add L4-only visibility redirect key if the L4-only
-//    key does not already exist.
-//    - e.g., 0:0=allow,0 -> add 0:80=allow,<proxyport> if 0:80 does not exist
-//      - this allows all traffic on port 80, but see step 5 below.
-// 3. Change all L3/L4 ALLOW keys on matching port that do not already redirect to
-//    redirect.
-//    - e.g, <ID1>:80=allow,0 -> <ID1>:80=allow,<proxyport>
-// 4. For each L3-only ALLOW key add the corresponding L3/L4 ALLOW redirect if no
-//    L3/L4 key already exists and no L4-only key already exists and one is not added.
-//    - e.g., <ID2>:0=allow,0 -> add <ID2>:80=allow,<proxyport> if <ID2>:80
-//      and 0:80 do not exist
-// 5. If a new L4-only key was added: For each L3-only DENY key add the
-//    corresponding L3/L4 DENY key if no L3/L4 key already exists.
-//    - e.g., <ID3>:0=deny,0 -> add <ID3>:80=deny,0 if <ID3>:80 does not exist
+//  1. Change existing L4-only ALLOW key on matching port that does not already
+//     redirect to redirect.
+//     - e.g., 0:80=allow,0 -> 0:80=allow,<proxyport>
+//  2. If allow-all policy exists, add L4-only visibility redirect key if the L4-only
+//     key does not already exist.
+//     - e.g., 0:0=allow,0 -> add 0:80=allow,<proxyport> if 0:80 does not exist
+//     - this allows all traffic on port 80, but see step 5 below.
+//  3. Change all L3/L4 ALLOW keys on matching port that do not already redirect to
+//     redirect.
+//     - e.g, <ID1>:80=allow,0 -> <ID1>:80=allow,<proxyport>
+//  4. For each L3-only ALLOW key add the corresponding L3/L4 ALLOW redirect if no
+//     L3/L4 key already exists and no L4-only key already exists and one is not added.
+//     - e.g., <ID2>:0=allow,0 -> add <ID2>:80=allow,<proxyport> if <ID2>:80
+//     and 0:80 do not exist
+//  5. If a new L4-only key was added: For each L3-only DENY key add the
+//     corresponding L3/L4 DENY key if no L3/L4 key already exists.
+//     - e.g., <ID3>:0=deny,0 -> add <ID3>:80=deny,0 if <ID3>:80 does not exist
```

Comment:
Unintended diff?

Reply:
Someone updated this comment section in v1.13 because 1.11 backport also has the weird reformat.